microsoft-edge/visual-studio-code/index.md

```diff
@@ -38,3 +38,13 @@ By adding the [Elements for Microsoft Edge](https://marketplace.visualstudio.com
 For more information, check out [this page](./elements-for-edge.md).
 
 ![GIF of the Elements for Edge VS Code extension in action!](./media/elements-for-edge.gif)
```

Comment:
Please replace image with a .png file or create a video. 
For videos, please refer to [Plan a video](https://review.docs.microsoft.com/help/contribute/contribute-video-plan?branch=master).

Reply:
This file isn't part of my requested changes, and I'm not the right point of contact for editing Elements for Edge contact. Let's hold on this for now and open a separate PR with the right person assigned?

Reply:
cc @erdraud--See the comment above for a future PR against Elements for Edge content. Doesn't need to be part of this PR.

Reply:
Moving request per peer suggestion and resolving.